src/libre/dialect/pcre/lexer.lx

```diff
@@ -167,9 +167,12 @@
 
 '(?' -> $flags .. ':' -> $sub, ')' -> $close {
 	'-' -> $negate;
+	'xx' -> $flag__unknown;  # xx (EXTENDED_MORE) is not yet supported
 	'i' -> $flag__insensitive;
 	's' -> $flag__single;
-	flags = $flag__insensitive | $flag__single;
+	'x' -> $flag__extended;
+	'n' -> $flag__ignore; # 'n' is the no auto capture flag, which we can ignore
```

Comment:
@silentbicycle heads up, I suppose we'll want to un-ignore this eventually, because having it as a no-op would be incorrect.